core/templates/dev/head/pages/exploration_player/StatsReportingService.js

```diff
@@ -46,15 +46,17 @@ oppia.constant('STATS_REPORTING_URLS', {
 });
 
 oppia.factory('StatsReportingService', [
-  '$http', '$interval', 'StopwatchObjectFactory', 'MessengerService',
-  'UrlInterpolationService', 'STATS_REPORTING_URLS', 'siteAnalyticsService',
-  'STATS_EVENT_TYPES', 'ExplorationContextService', 'PAGE_CONTEXT',
-  'DEFAULT_OUTCOME_CLASSIFICATION', 'PlaythroughService', 'ENABLE_PLAYTHROUGHS',
+  '$http', '$interval', 'ExplorationContextService', 'MessengerService',
+  'PlaythroughService', 'siteAnalyticsService', 'UrlInterpolationService',
+  'StopwatchObjectFactory', 'DEFAULT_OUTCOME_CLASSIFICATION',
+  'ENABLE_PLAYTHROUGH_RECORDING', 'PAGE_CONTEXT', 'STATS_EVENT_TYPES',
+  'STATS_REPORTING_URLS',
   function(
-      $http, $interval, StopwatchObjectFactory, MessengerService,
-      UrlInterpolationService, STATS_REPORTING_URLS, siteAnalyticsService,
-      STATS_EVENT_TYPES, ExplorationContextService, PAGE_CONTEXT,
-      DEFAULT_OUTCOME_CLASSIFICATION, PlaythroughService, ENABLE_PLAYTHROUGHS) {
+      $http, $interval, ExplorationContextService, MessengerService,
+      PlaythroughService, siteAnalyticsService, UrlInterpolationService,
+      StopwatchObjectFactory, DEFAULT_OUTCOME_CLASSIFICATION,
```

Comment:
Ah, I meant other way around (to keep them alphabetical).

Reply:
Ahh, I see. Done.